code/style.css

```diff
@@ -1,7 +1,428 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  font-family: 'Open Sans Condensed', sans-serif;
+  background-color: #F9F3F3;
+  margin: 0;
+}
+
+/* ------HEADER SECTION STYLING ----- */
+
+header {
+  overflow: hidden;
+  font-size: 1rem;
+}
+
+.logo {
+  position: fixed;
+  top: 3%; 
+  left: 2%;
+  width: 40px;
+}
+
+.hero-slide {
+  position: relative;
+}
+
+.hero-slide p {
+  font-family: 'Dancing Script', cursive;
+  position: absolute;
+  top: 60%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 16px;
+  color: #ffffff;
+  margin: 0px;
+}
+
+.contact-link {
```

Comment:
This link is really small on bigger devices and kind of hard to see up there in the corner. I would make that link a bit more in line with the logo and a bit bigger (or another color?